### PR TITLE
scalastyle: Update binary and config to 0.6.0

### PR DESCRIPTION
**Pull request by jawshooah**

### CODE REVIEW

File: Library/Formula/scalastyle.rb [hunk 1]
```diff
@@ -2,24 +2,24 @@
 
 class Scalastyle < Formula
   homepage "http://www.scalastyle.org/command-line.html"
-  url "https://oss.sonatype.org/content/repositories/releases/org/scalastyle/scalastyle-batch_2.10/0.5.0/scalastyle-batch_2.10-0.5.0-distribution.zip"
-  version "0.5.0"
-  sha1 "bf009e0c4d48e5effce5113f5feb3da626ca1248"
+  url "https://oss.sonatype.org/content/repositories/releases/org/scalastyle/scalastyle_2.10/0.6.0/scalastyle_2.10-0.6.0-batch.jar"
+  version "0.6.0"
```
Comment (bfontaine) [inline Library/Formula/scalastyle.rb]: You should remove this line, Homebrew already extract the version from the URL (this is why the build failed).

Comment (jawshooah) [inline Library/Formula/scalastyle.rb]: Fixed. Does the same apply to the config file? When I ran `brew upgrade scalastyle` without the `version` clause for the config, it claimed that `scalastyle_config.xml` was already downloaded, and the checksum failed. Adding the `version` fixed it.

**Comment by jawshooah** [comment]
@bfontaine Fixed. Does the same apply to the config file? When I ran `brew upgrade scalastyle` without the `version` clause for the config, it claimed that `scalastyle_config.xml` was already downloaded, and the checksum failed. Adding the `version` fixed it.

**Comment by bfontaine** [comment]
@jawshooah Great. No it doesn't apply to the config file here (maybe someone like @mikemcquaid can confirm what I’m saying).

**Comment by mikemcquaid** [comment]
@jawshooah Yeh, without the version it'll assume it's the same file as the old version and the URL has changed.

**Pull request closed without merging**